x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -1195,3 +1195,30 @@ function deepMergeVars(original: any, override: any): any {
 
   return result;
 }
+
+export async function incrementPackageName(
+  soClient: SavedObjectsClientContract,
+  packageName: string
+) {
+  // Fetch all packagePolicies having the package name
+  const systemPackagePolicyData = await packagePolicyService.list(soClient, {
```

Comment:
(nit) In terms of naming, I found the `systemPackagePolicyData` name a bit confusing here, since the implementation allows for any package name. I initially thought this was doing something w/ a special case for the `system` package. Maybe we can simply discard the "system" prefix in this variable name?

Reply:
You're right, I had the `system` in the name in the first round of implementation and then I forgot to change it. I'll fix it for better clarity.